queries/d/highlights.scm

```diff
@@ -0,0 +1,265 @@
+;; Misc
+
+[
+  (line_comment)
+  (block_comment)
+  (nesting_block_comment)
+] @comment
+
+[
+  "(" ")"
+  "[" "]"
+  "{" "}"
+] @punctuation.bracket
+
+[
+  ","
+  ";"
+  "."
+] @punctuation.delimiter
+
+[
+  ".."
+  "$"
+] @punctuation.special
+
+;; Constants
+
+[
+  "__FILE_FULL_PATH__"
+  "__FILE__"
+  "__FUNCTION__"
+  "__LINE__"
+  "__MODULE__"
+  "__PRETTY_FUNCTION__"
+] @constant.macro
+
+(string_literals) @string
+
+; Don't highlight token strings as strings
+(token_string_tokens) @none
+
+(character_literal) @character
+
+(integer_literal) @number
+
+(float_literal) @float
+
+[
+  "true"
+  "false"
+] @boolean
+
+;; Functions
+
+(func_declarator
+  (identifier) @function
+)
+
+[
+  "__traits"
+  "__vector"
+  "assert"
+  "is"
+  "mixin"
+  "pragma"
+  "typeid"
+] @function.builtin
+
+(import_expression
+  "import" @function.builtin
+)
+
+(parameter
+  (var_declarator
+    (identifier) @parameter
+  )
+)
+
+(function_literal
+  (identifier) @parameter
+)
+
+(constructor
+  "this" @constructor
+)
+
+(destructor
+  "this" @constructor
+)
+
+;; Keywords
+
+[
+  "case"
+  "default"
+  "else"
+  "if"
+  "switch"
+] @conditional
+
+[
+  "do"
+  "for"
+  "foreach"
+  "foreach_reverse"
+  "while"
+] @repeat
+
+[
+  "__gshared"
+  "__parameters"
+  "abstract"
+  "alias"
+  "align"
+  "asm"
+  "auto"
+  "body"
+  "break"
+  "class"
+  "const"
+  "continue"
+  "debug"
+  "deprecated"
+  "do"
```

Comment:
`do` is already in repeat

Reply:
continue probably also

Reply:
If `continue` should be in `@repeat`, should `break` be there too?

Reply:
yes